config/src/main/java/com/alibaba/nacos/config/server/model/ConfigHistoryInfoPair.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.nacos.config.server.model;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * extensible {@link ConfigHistoryInfo}.
+ *
+ * @author dirtybit
+ */
+public class ConfigHistoryInfoPair extends ConfigHistoryInfo implements Serializable {
```

Comment:
create a new  model instead ,do not extends ConfigHistoryInfo , ConfigHistoryInfo represent persist model,not a business model.

Reply:
the content in ConfigHistoryInfo is not clear for different  optype

Reply:
> create a new model instead ,do not extends ConfigHistoryInfo , ConfigHistoryInfo represent persist model,not a business model.

ok